projects/soulswap/helper/getBlock.js

```diff
@@ -0,0 +1,18 @@
+const sdk = require('@defillama/sdk')
+const retry = require('async-retry')
+const axios = require('axios');
+
+async function getBlock(timestamp, chain, chainBlocks) {
+    if(chain === 'celo'){
```

Comment:
why fork this file?

Reply:
thanks for letting me know. I have now deleted redundant files and updated paths to pull from helper global import.